_includes/tutorials/finding-distinct/kstreams/code/build.gradle

```diff
@@ -32,9 +32,9 @@ apply plugin: "com.github.johnrengelman.shadow"
 dependencies {
     compile "org.apache.avro:avro:1.8.2"
     implementation "org.slf4j:slf4j-simple:1.7.26"
-    implementation "org.apache.kafka:kafka-streams:2.3.0"
+    implementation "org.apache.kafka:kafka-streams:2.4.1"
```

Comment:
With this change I get the following when trying to run the app at step 6

```
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/kafka/clients/admin/Admin
	at org.apache.kafka.streams.KafkaStreams.<init>(KafkaStreams.java:557)
	at io.confluent.developer.FindDistinctEvents.runRecipe(FindDistinctEvents.java:235)
	at io.confluent.developer.FindDistinctEvents.main(FindDistinctEvents.java:225)
Caused by: java.lang.ClassNotFoundException: org.apache.kafka.clients.admin.Admin
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	... 3 more
```
Maybe just revert to `2.3.0` for now?

Reply:
Yes, no time to investigate it right now... reverted to `2.3.0`.